src/dfm/description/gen_description.py

```diff
@@ -0,0 +1,120 @@
+import os
+import sys
+import time
+from typing import List
+from datasets import load_dataset
+
+
+import spacy
+
+sys.path.append(".")
+
+from dfm.description.description_patterns import (
+    female_gendered_terms,
+    male_gendered_terms,
+    occupation_pattern_list,
+    danish_adult_words,
+    get_muslim_name_patterns,
+    get_gender_name_patterns,
+)
+
+from dfm.description.match_counter import MatchCounter
+
+
+def create_patterns() -> List:
+    """Generates the patterns we've selected for the present analyses.
+
+    Returns:
+        List: List of spacy pattern containers.
+    """
+    any_token_pattern = [{"tokens": [{"TEXT": {"REGEX": ".+"}}]}]
+
+    # Religion
+    religion_patterns = [
+        {"atheist": [{"LOWER": {"REGEX": "ath{0,1}eis.*"}}]},
+        {"buddhist": [{"LOWER": {"REGEX": "buddh{0,1}.*"}}]},
+        {"christian": [{"LOWER": {"REGEX": "kriste.*"}}]},
+        {"hindu": [{"LOWER": {"REGEX": "hindu.*"}}]},
+        {"muslim": [{"LOWER": {"REGEX": "muslim.*"}}]},
+        {"jew": [{"LOWER": {"REGEX": "jødi.*"}}]},
+        {"jew": [{"LOWER": {"REGEX": "(?!jødeskæg)jøde.*"}}]},  # Jødeskæg == Stueplante
```

Comment:
I always advice against using regexes when term lists will do. Regexes might be shorter to write but are more difficult to read and include you didn't consider. My dad last name is `kristensen`, but I can assure you he's no christian.

Reply:
I looks like we could transfor all data into term lists, and thus get rid of this custom function and make it easy to add new data (simply include a new list of terms in a data folder)